### DISCUSSION TITLE
prefix arguement for align regexp

### DISCUSSION THREAD

**Discussion by Inc0n** [condy0919/emacs-newbie#9] [Ideas]
For `align-regexp` I would like to suggest, to mention the prefix argument interface that this command has in the built in mode guide, where it allows the user to control the behaviour of alignment on a more finer degree.

**Comment by condy0919** [comment]
Maybe I should add a link to https://emacs-china.org/t/topic/13464/10

**Reply by Inc0n** [reply]
Yh sure that would do
